app/models/grda_warehouse/client_pii.rb

```diff
@@ -0,0 +1,66 @@
+# mask PII attributes on the client record
+class GrdaWarehouse::ClientPii
+  attr_reader :user, :client
+
+  def initialize(user:, client:)
+    @user = user
+    @client = client
+  end
+
+  def first_name
+    name_part(client.first_name.presence)
+  end
+
+  def last_name
+    name_part(client.last_name.presence)
+  end
+
+  def middle_name
+    name_part(client.middle_name.presence)
+  end
+
+  def full_name
+    [first_name, middle_name, last_name].compact.join(' ').presence
+  end
+
+  def brief_name
+    [first_name, last_name].compact.join(' ').presence
+  end
+
+  DATE_FMT = '%m/%d/%Y'.freeze
```

Comment:
On the warehouse side, we've always used `Aug 30, 2021` as the date format.  It's non-US standard, but I don't think we should enforce a different format here.